src/components/MaterialUI/DataDisplay/PredefinedElements/Translations.test.js

```diff
@@ -0,0 +1,162 @@
+import React from "react";
+import { mount } from "enzyme";
+import sinon from "sinon";
+import Collapse from "@material-ui/core/Collapse";
+import Typography from "@material-ui/core/Typography";
+import Icon from "../Icon";
+import { TestWrapper, createMuiTheme, extractMessages } from "../../../../utils/testUtils";
+import InputBase from "@material-ui/core/InputBase";
+import Translations, { compareTranslateStrings } from "./Translations";
+import sharedMessages from "../../../../sharedMessages";
+import { buildUrl } from "../../../../utils/buildUrl";
+
+const messages = extractMessages(sharedMessages);
+
+describe("Translations ", () => {
+	const cultures = ["en-CA", "en-US", "fr-CA"];
+	const defaultCulture = "en-US";
+
+	const theme = createMuiTheme();
+
+	it("Renders Translations  correctly", () => {
+		const component = (
+			<TestWrapper stylesProvider muiThemeProvider={{ theme }} intlProvider={{ messages }}>
+				<Translations cultures={cultures} defaultCulture={defaultCulture} />
+			</TestWrapper>
+		);
+
+		const expected = (
+			<TestWrapper stylesProvider muiThemeProvider={{ theme }}>
+				<div>
+					{
+						<div key={"en-US"}>
+							<div>
+								<div>
+									<label>{"en-US"}</label>
+									<InputBase />
+								</div>
+							</div>
+						</div>
+					}
+					<Collapse in={false} timeout="auto">
+						{[
+							<div key={"en-CA"}>
+								<div>
+									<div>
+										<label>{"en-CA"}</label>
+										<InputBase />
+									</div>
+								</div>
+							</div>,
+							<div key={"fr-CA"}>
+								<div>
+									<div>
+										<label>{"fr-CA"}</label>
+										<InputBase />
+									</div>
+								</div>
+							</div>,
+						]}
+					</Collapse>
+					<div>
+						<Icon color="primary" id="chevron-down" />
+						<Typography color="primary" children={"Show more languages"} />
+					</div>
+				</div>
+			</TestWrapper>
+		);
```

Comment:
it's not what you should expect here. you are expecting CollapsableList with correct props